src/czt.jl

```diff
@@ -1,7 +1,7 @@
 export czt, iczt
 
 """
-    czt_1d(xin , scaled , d)
+    czt_1d(xin , scaled , d; kill_wrap=false, pad_value=zero(eltype(xin)))
```

Comment:
maybe different wording than `kill_wrap`. `remove_wrap`?

Reply:
agreed. I changed it.